simulators/main/event.py

```diff
@@ -0,0 +1,318 @@
+import json, logging
+from datetime import datetime
+from random import randint, uniform, choices
+from cumulocityAPI import CumulocityAPI
+from task import Task, PeriodicTask
+import interface
+
+cumulocityAPI = CumulocityAPI()
+log = logging.getLogger("events generation")
+
+
+class Event(interface.MachineType):
+    def __init__(self, model, shiftplans) -> None:
+        self.model = model
+        self.shiftplans = shiftplans
```

Comment:
Why are shiftplans related to Event or Measurements? I would think they should be part of the eventsimulator

Reply:
1. This Event class is the Event simulator class you mentioned.